low-level/framework/sspl_ll_d

```diff
@@ -408,6 +412,66 @@ def main(conf_reader, systemd_support):
     except Exception as ex:
         logger.exception(ex)
 
+def _tune_modules_for_env(modlist):
+    """Enable modules based on environment"""
+    if not modlist:
+        return modlist
+
+    # current deployment env declared here, Extend checks based on need in future
+    # like server_type, etc
+
+    NODE_KEY = 'nodes'
+    SERVER_PER_NODE_KEY = 'servers_per_node'
+    STORAGE_TYPE_KEY = 'storage_type'
+    SERVER_TYPE_KEY = 'server_type'
+
+    STORAGE_TYPE = 'physical'
+    SERVER_TYPE = 'rbod'
+    SERVER_PER_NODE = 2
+    NODES = 1
+
+    info_err = None
+
+    try:
+        util = Utility()
+    except Exception as err:
+        logger.error(f'Can not instantiate Utility class, Error:{err}')
+
+    try:
+        if not util:
+            # IF for any reason util class fails to instantiate, without blocking the execution
+            # Go ahead with excution using subprocess module here directly
+            process = subprocess.Popen(['sudo', '/usr/local/bin/provisioner', 'get_setup_info'], \
+                                 shell=False, stdout=subprocess.PIPE)
+            _result, _ret_code = process.communicate()[0], process.returncode
+        else:
+            _result, _err, _ret_code = util.execute_cmd(['sudo', '/usr/local/bin/provisioner', 'get_setup_info'])
```

Comment:
I saw, earlier it was /usr/bin.. But I checked
[root@ssc-vm-0917 cortx-monitor]# which provisioner
/usr/local/bin/provisioner
So, I changed the path.